CHANGELOG-3.3.md

```diff
@@ -7,6 +7,8 @@ in 3.3 minor versions.
 To get the diff for a specific change, go to https://github.com/symfony/symfony/commit/XXX where XXX is the change hash
 To get the diff between two versions, go to https://github.com/symfony/symfony/compare/v3.3.0...v3.3.1
 
+ * feature #23499 [Workflow] add guard is_valid() method support
```

Comment:
Your need to update [this changelog](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Workflow/CHANGELOG.md)

The current one is updated automatically.